source/ui/acquaman/AMGenericStepScanConfigurationView.cpp

```diff
@@ -0,0 +1,450 @@
+#include "AMGenericStepScanConfigurationView.h"
+
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+#include <QGroupBox>
+#include <QScrollArea>
+#include <QCheckBox>
+#include <QButtonGroup>
+
+#include "ui/AMTopFrame.h"
+#include "beamline/AMBeamline.h"
+
+AMGenericStepScanConfigurationView::AMGenericStepScanConfigurationView(AMGenericStepScanConfiguration *configuration, QWidget *parent)
+	: AMScanConfigurationView(parent)
+{
+	configuration_ = configuration;
+
+	AMTopFrame *frame = new AMTopFrame("BioXAS Commissioning Tool");
```

Comment:
In general, no.  However, I have given a task to @dleacock to make a convenience view constructor for the app controller that automatically builds top frames and the other necessary pieces.  We'll fix what you've tagged here when he finishes that.